lang/java/reef-tang/tang/src/main/java/org/apache/reef/tang/formats/CommandLine.java

```diff
@@ -204,7 +207,7 @@ public static ConfigurationBuilder parseToConfigurationBuilder(final String[] ar
     try {
       commandLine = new CommandLine().processCommandLine(args, argClasses);
     } catch (final IOException e) {
-      // processCommandLine() converts ParseException into IOException. This reverts that to make exception handling
+      // processCommandLine() converts IOException into ParseException. This reverts that to make exception handling
```

Comment:
`processCommandLine` catches `ParseException` and throws `IOException`. This piece of code does the contrary. The old comment seems to be correct?